llm_studio/app_utils/sections/dataset.py

```diff
@@ -324,7 +354,7 @@ async def dataset_import(
                 ui.textbox(
                     name="dataset/import/kaggle_command",
                     label="Kaggle API command",
-                    value=default_cfg.kaggle_command,
+                    placeholder="kaggle competitions download -c dataset",
```

Comment:
the placehold will not be used though, or will it?
user needs to enter something there then